.github/workflows/python-package-conda.yaml

```diff
@@ -32,58 +43,73 @@ jobs:
 
       - name: Run docformatter
         shell: bash
-        run: pre-commit run docformatter --all-file
+        run: pre-commit run docformatter --all-files
 
       - name: Run pydocstyle
         shell: bash
-        run: pre-commit run pydocstyle --all-file
-
-  mypy:
-    if: always()
-    runs-on: ubuntu-latest
-    timeout-minutes: 2
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-
-      - uses: actions/setup-python@v4
-        with:
-          python-version: "3.9"
-          cache: "pip"
-
-      - name: Install Dependencies
-        run: |
-          pip install --upgrade pip
-          pip install -r requirements.txt
+        run: pre-commit run pydocstyle --all-files
 
       - name: Run mypy
         shell: bash
-        run: pre-commit run mypy --all-file
+        run: pre-commit run mypy --all-files
+   
+      - name: Run pytest with code coverage
+        run: |
+          pytest --junitxml pytest.xml --cov=. --cov-report=term-missing --cov-report=xml:coverage.xml --cov-branch | tee pytest-coverage.txt
 
-  build-linux:
-    runs-on: ubuntu-latest
-    strategy:
-      max-parallel: 5
+      - name: Upload Coverage Results
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage-results
+          path: coverage.xml
+        if: ${{ always() }}
 
-    steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python 3.9
-      uses: actions/setup-python@v4
-      with:
-        python-version: 3.9
-        cache: "pip"
-
-    - name: Install Dependencies
-      run: |
-        pip install --upgrade pip
-        pip install -r requirements.txt
-   
-    - name: Run tests
-      run: |
-        pytest tests --cov=./ --cov-report=xml
-
-    - name: "Upload coverage to Codecov"
-      uses: codecov/codecov-action@v2
-      with:
-        token: ${{ secrets.CODECOV_TOKEN }}
\ No newline at end of file
+      - name: Upload Unit Test Results
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: unit-test-py39
+          path: pytest.xml
+
+      - name: Upload Unit Test Results
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: unit-test-py39
+          path: pytest-coverage.txt
+          publish-test-results:
+  publish-test-results:    
```

Comment:
I don't understand the necessity for this as this workflow is triggered on a push, where will the comment appear?

Reply:
Ya probably this is not needed.